.github/workflows/build.yml

```diff
@@ -113,7 +113,7 @@ jobs:
         # --scan:         Publish the build scan. This will only work on PRs from apache/kafka and trunk
         # --no-scan:      For public fork PRs, we won't attempt to publish the scan
         run: |
-          ./gradlew --build-cache --info $SCAN_ARG check siteDocTar -x test
+          ./gradlew --build-cache --info $SCAN_ARG check siteDocTar releaseTarGz -x test
```

Comment:
releaseTarGz includes siteDocTar, so I think we can exclude siteDocTar here

Reply:
hi @mumrah , I have excluded `siteDocTar`